### ISSUE TITLE
Customizable pop-up box

### ISSUE THREAD

**Issue by mark-btmt** [OktaSecurityLabs/passprotect-chrome#3]
Hi Randall!

Thanks for making this extension! Appreciate the work put into it! Great initiative!

I'm wondering if there's a way to customize the contents of the pop-up box that shows up. I looked through my local chrome extension's files, and did not find any of the relevant contents in there, so I couldn't modify them locally. Are they being fetched during runtime?

If the pop-up box contents were available locally, I could modify them directly to make it work. I'm not sure if there is a different way of allowing this sort of customization.

Customizing the message shown to users would be valuable esp. in a corporate setting where we would be able to provide some more context, and give users the relevant email address to contact in case they need more info. That could lead to an increased rate of adoption of the extension.

Let me know what you think! Thank you in advance!

**Comment by RuchiraKonda-okta** [comment]
Hey  Hi !

Thanks for the suggestion. It's a great idea to have a customizable pop-up  by adding more relevant information .Do let me know if this is something you wanted or any CSS that should be added to the embedded pop-up box .Thanks !
![Screen Shot 2019-07-29 at 11 31 50 AM](https://user-images.githubusercontent.com/53058109/62075837-07f64b00-b1fb-11e9-97c3-0b0c8a4e2a12.png)
![Screen Shot 2019-07-29 at 11 32 34 AM](https://user-images.githubusercontent.com/53058109/62075838-07f64b00-b1fb-11e9-8be8-4dacaf5f94c1.png)

**Comment by mark-btmt** [comment]
Thanks for the response @RuchiraKonda-okta 

Yeah the screenshot looks good, except I'd prefer having the ability to change the entire contents of that pop-up box. 
Also one thing worth noting is that I'm not using it for a single website, I'm deploying it on org laptops' browsers so it is enabled everywhere as a chrome extension. Any idea if I can customize it in this case?
Thanks!

**Comment by RuchiraKonda-okta** [comment]
Hi @mark-btmt 
Great idea ! If there is customization, it would reflect a customized message.
 I'm wondering If there is no customization what would you want to be displayed ? the initial default warning message ?

**Comment by mark-btmt** [comment]
Yeah exactly, I think the default warning message is great! So if there is no org-wide customized message, showing the default one is perfect.
Let me know if I can help out in any way.
Thank you!

**Comment by RuchiraKonda-okta** [comment]
@mark-btmt  Sounds excellent! It would be great if you could let me know if any styling should be applied to the embedded user option pop up box?
Thank you !

**Comment by mark-btmt** [comment]
The only styling I would personally like is supporting images, specifically company logos. I think users may be less likely to dismiss/ignore a pop-up box if it is tailored to them and has a company logo in it.

By adding the company logo (and modifying the message content to include an org email address employees can reach out to) the pop-up box would become much more relevant and seemingly useful overall.

Thanks so much for your understanding and quick responses!

**Comment by RuchiraKonda-okta** [comment]
@mark-btmt  Thank you for your suggestions. Would something like this work ?
![Screen Shot 2019-08-13 at 4 16 58 PM](https://user-images.githubusercontent.com/53058109/62983922-037a8680-bde6-11e9-87e0-ee56eb77d5c4.png)

**Comment by RuchiraKonda-okta** [comment]
@mark-btmt  Thank you so much for your suggestions.Would something like this work?
Thanks !
![Screen Shot 2019-08-13 at 4 16 58 PM](https://user-images.githubusercontent.com/53058109/62983992-489eb880-bde6-11e9-8e3c-63ff04635362.png)

**Comment by mark-btmt** [comment]
Yeah this looks great!
How would I include images? Does it accept markdown or would it render an image that I link?
Also how would the process look if we're trying to push this out to everyone using it within our org? i.e. if we want all employees to see the same custom message
